bn-models/src/models/trips-api/catches.ts

```diff
@@ -13,6 +13,22 @@ export enum errorType {
   showStopper = 'showStopper',
 }
 
+export enum gearType {
```

Comment:
Believe a better place for this would be in the gear.ts file where the model for the couch lookup document is defined that way catches.ts doesn't just become a dumping ground for enums. Also is this an EM gear type? If it isn't there's a gear-types-ts file in models/_lookups, believe that would be a better place for this enum. Also if it is just for EM thinking it might be good to have a folder named EM where we put all the lookup docs there. Right now trips-api folder is a bit of a confusing name so think it'd be good to create an EM folder in addition

Reply:
Implementing